Scripts/script-CommonServerPython.yml

```diff
@@ -254,7 +254,7 @@ script: |-
   def formatCell(data, is_pretty=True):
       l_format = '\n' if is_pretty else ''
       if type(data) in STRING_TYPES:
-          return data
+          return data.replace("|","\\|")
```

Comment:
this will only solve the "|" character, what about the rest of them? the issue mentions other characters as well

Reply:
I know... but I don't think we want to escape other chars, and I don't want to do many changes